packages/element-templates-json-schema-shared/src/defs/condition.json

```
@@ -9,19 +9,21 @@
         "type": {
           "$id": "#/condition/type",
           "const": "simple",
-          "description": "The type of the condition.",
+          "description": "Defines the type of the condition (always \"simple\").",
           "default": "simple"
         },
         "property": {
           "$id": "#/condition/property",
           "type": "string",
-          "description": "The id of the property to check."
+          "description": "Unique identifier of the property. Used, for example, to reference the property in conditions."
```

This one is incorrect. We use `property` key to denote the `id` of another property that the condition depends on. 
```suggestion
          "description": "Reference to the property whose value is used to evaluate the condition."
```
@lmbateman can you have a look at the suggestion?